### PR TITLE
2892 i18n blog tags

### LINKED ISSUE #2892: Blog tags for non-English blog posts
Work through how to organize the tags for non-English blog posts. Currently, there is only one file that controls the blog post tags.  https://github.com/OpenLiberty/blogs/blob/prod/blog_tags.json.

Should all blog posts have their tags in this single file or should there be `blog_tags` file for each language?

### PR DESCRIPTION
**Pull request by natalie-bernhard**
## What was changed and why?
Non-english blog tags are now enabled, and some optimization was done to reduce our front-end JS and increase the work done by templates during build time.

There are a lot of changes in this PR, so please feel free to reach out with any questions you may have.

## Tested using browser:
- [X] Firefox (Desktop)
- [ ] Safari (Desktop)
- [X] Chrome (Desktop)

### CODE REVIEW

File: src/main/content/_assets/js/blog.js [hunk 1]
```diff
@@ -180,6 +137,7 @@ var blog = function(){
             for(var i = 0; i < query_params.length; i++){
                 if(query_params[i].indexOf('search=') === 0 || query_params[i].indexOf('search!=') === 0) {
                     var tag_name;
+                    debugger;
```
Comment (steven1046) [inline src/main/content/_assets/js/blog.js]: We can remove this

File: src/main/content/_assets/js/blog.js [hunk 2]
```diff
@@ -230,23 +192,29 @@ var blog = function(){
...
+    blog.init();
+});
+
 $(document).ready(function() {
     blog.adjustWhiteBackground();
-    blog.getTags(function () {
-        blog.init();
-    });
+    blog.init();
+    $(document).on("click keypress", ".featured_tag, .blog_tag, .blog_tags_container > p", function(e){
```
Comment (steven1046) [inline src/main/content/_assets/js/blog.js:215]: Does this work for keyboard navigation with enter?

File: src/main/content/_i18n/zh-Hans.yml [hunk 3]
```diff
@@ -230,19 +230,43 @@ blog:
   twitter_or_subscribe: 或订阅我们的
   feed: 订阅源。
   featured_tags: 特色标签
-  no_results_found: No results found.
-  see: See
-  all_blogs: all blogs.
-  filtered_by_tag: Filtered by tags
+  no_results_found: 找不到任何结果
+  see: null
```
Comment (steven1046) [inline src/main/content/_i18n/zh-Hans.yml]: What does the null do on the client side?

File: src/main/content/_assets/js/blog.js (hunk 2 above)
Comment (natalie-bernhard) [reply src/main/content/_assets/js/blog.js:215]: Yes

**Review by steven1046 — APPROVED** [review]

**Pull request merged**